### DISCUSSION TITLE
What is the structure of the team?

### DISCUSSION THREAD

**Discussion by marsrobertson** [hypercerts-org/hypercerts#433] [General]
I tried to DYOR but it is non-obvious

# Old repo

https://github.com/Network-Goods/hypercerts-pilot/graphs/contributors

<img width="611" alt="image" src="https://user-images.githubusercontent.com/44369284/223184771-06a3fa0c-4c62-4dc7-9e47-da0f3ee5717d.png">

# New repo

https://github.com/hypercerts-org/hypercerts/graphs/contributors

<img width="613" alt="image" src="https://user-images.githubusercontent.com/44369284/223184875-224818a8-b72a-424a-8404-2a19e7d9fe04.png">

# There is also https://www.raidguild.org/

<img width="401" alt="image" src="https://user-images.githubusercontent.com/44369284/223185340-69288816-f378-4032-97a5-e3013dc57008.png">

# There is also https://supermodular.xyz/

<img width="1204" alt="image" src="https://user-images.githubusercontent.com/44369284/223185618-08ecf302-a301-4536-8cad-055391a3094c.png">

# What is the current hiring strategy?

From the Funding the Commons talk: https://testnet.hypercerts.org/docs/further-resources

<img width="367" alt="image" src="https://user-images.githubusercontent.com/44369284/223186174-a582ff21-7f77-48cb-a74d-5b26cca7fe21.png">

**Comment by ryscheng** [comment]
There are a bunch of amazing contributors at this point, if you're looking for historical contributions, don't forget the other repos labeled hypercerts-* under https://github.com/orgs/Network-Goods/repositories

With the new Hypercerts Foundation, we are actively discussing how to best structure the team to be most effective. If you have thoughts, please feel free to share what you think is working / not working. We'd love to hear it!

**Comment by marsrobertson** [comment]
> don't forget the other repos

I've completely missed the fact that there are other repos in that org :)

The reason for asking about the team structure is my personal situation:
- I have been working on a very similar project
- I did not succeed as a solo-founder
- As a consequence, I started looking for a job...
- ...not just any job but something truly meaningful
- I've been pitching the idea of measuring impact and I would be perfect in BizDev / Marketing / DevRel / Comms type of work
- Wearing many hats - I'm also comfy as a coder - [top 3% on StackOverflow](https://stackoverflow.com/users/775359/mars-robertson?tab=topactivity)

Proof of work: https://github.com/hypercerts-org/hypercerts/issues/431 _(some of the research coming from my project)_

I would love to discuss the IE (impact evaluation) process as well as the negative impact - more details in the issue linked above.

**Comment by marsrobertson** [comment]
So this just happened... In the spirit of transparency.

<img width="863" alt="image" src="https://user-images.githubusercontent.com/44369284/223432221-86a6a1cd-6d6c-4747-b92f-711291382921.png">


### Cover letter

[2023-03-07 Hypercert Cover Letter.pdf](https://github.com/hypercerts-org/hypercerts/files/10909686/2023-03-07.Hypercert.Cover.Letter.pdf)

### CV

Don't want to doxx myself too much, you know where to find me.

**Comment by marsrobertson** [comment]
Related.

Asked about the structure of the team.

I'm genuinely curious about the legal structure, related to #435 

https://twitter.com/hypercerts/status/1628099594881060864

<img width="529" alt="image" src="https://user-images.githubusercontent.com/44369284/224051441-265a6fa9-b7ab-4ecf-81be-993459a12b19.png">

https://hypercerts.notion.site/Introducing-the-Hypercerts-Foundation-d956203fe0fc4792980da138015e770a

<img width="479" alt="image" src="https://user-images.githubusercontent.com/44369284/224051912-2c413568-42b4-4251-b767-a30ef6d98cfb.png">

# Picking jurisdiction like a menu in a restaurant

https://otonomos.com/

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/44369284/224052162-b0c7e13d-c9f6-4c4e-9c91-d13a494cf1f7.png">

They are knowledgeable when comes to legal structures that are compliant with the on-chain and legacy 3D world: https://otonomos.substack.com/p/the-otoco-foundation-a-decentralized


<img width="717" alt="image" src="https://user-images.githubusercontent.com/44369284/224052409-6a24f863-6ff1-45fd-893a-2ba034f95c26.png">
